src/eager/test_models/mnist_fc.py

```diff
@@ -0,0 +1,45 @@
+from __future__ import print_function
```

Comment:
I suggest having one main test folder. there can be subfolder to organization, but a single test file so build scripts can handle them easily

Reply:
👍 A lot of this still needs cleaning up and organizing, but we want to get the two separate trees (ORTModule + eager) into the same repo to start. I will consolidate tests in a subsequent PR.